code/drasil-data/lib/Data/Drasil/Concepts/Computation.hs

```diff
@@ -20,24 +20,24 @@ modCalcDesc :: Sentence -> ConceptChunk
 modCalcDesc = dccWDS "modCalcDesc" (cn' "calculation")
 
 -- | Collects all computing-related named chunks (not concept-level yet).
-compcon :: [NamedChunk]
+compcon :: [IdeaDict]
 compcon = [application, computer, structure, dataStruct, dataStruct', dataType, dataType', 
   inDatum, outDatum, inParam, inVar, inValue, inQty, computerLiteracy, computerApp]
 
-application, computer, structure :: NamedChunk
+application, computer, structure :: IdeaDict
 os :: CI
 ------------------------------------------------------------------------------------
---  NC      |     |      id       |       term             |  abbreviation | domain
+-- IdeaDict |     |      id       |       term             |  abbreviation | domain
 -------------------------------------------------------------------------------------s
-application = nc   "application"      (cn' "application") 
-computer    = nc   "computer"         (cn' "computer") 
-structure   = nc   "structure"        (cn' "structure")         
+application = nc   "application"      (cn' "application")      Nothing
```

Comment:
This is a symptom that `nc` was incorrectly changed: it should assume `Nothing` for that argument, so that all these calls do NOT have to be changed.